app/routes/game.js

```diff
@@ -41,27 +41,12 @@ export async function draw(req, res) {
     return res.send(`Deck ${deckName} does not exist!`);
```

Comment:
Yep, that's a legit warning. I wrote it, so shame on me. Go ahead and remove the `deckName` variable from this return statement, or spend some time investigating what kind of sanitization is necessary here.

Reply:
@sbenthall This has now been addressed on `master`